documentation/guide-beanmapping.asciidoc

```diff
@@ -19,57 +19,17 @@ public UserEto mapUser(UserEntity source) {
 ----
 
 Therefore we are using a `BeanMapper` for this purpose that makes our lives a lot easier.
+There are several bean mapping frameworks with different approaches.
 
-== Bean-Mapper Dependency
-To get access to the `BeanMapper` we have to use either of below dependency in our POM:
+For a link:spring.asciidoc[devon4j-spring] application we recommend Orika, follow link:spring/guide-beanmapping-spring.asciidoc[Spring] for an introduction to Orika and Dozer in a devon4j-spring context application.
```

Comment:
I would prefer an other name for the second link. Just 'Spring' is not this meaningful